src/harness/testcases/WINNF_FT_S_HBT_testcase.py

```diff
@@ -0,0 +1,133 @@
+  def setUp(self):
+    self._sas, self._sas_admin = sas.GetTestingSas()
+    self._sas_admin.Reset()
+
+  def tearDown(self):
+    pass
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_HBT_9(self):
+    """ Array request, of three cbsds,SAS terminates the Grant for the one
+    that impacts incumbent activity.
+
+    successful heartbeat for the grants of two cbsds that don't impact
+     the incumbent and failed with responseCode 500 for the third one.
+    """
+    # STEP 1
+    # register three devices
+    # load  CBSD data
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json'))) 
+    device_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    device_e = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_e.json')))
+    request = {'registrationRequest': [device_a, device_c, device_e]}
+    response = self._sas.Registration(request)['registrationResponse']
+    cbsd_ids = []
+    for resp in response:
+      self.assertEqual(resp['response']['responseCode'], 0)
+      cbsd_ids.append(resp['cbsdId'])
+    del request, response
+
+    # load and set Grants data
+    grant_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_a['cbsdId'] = cbsd_ids[0]
+    grant_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_c['cbsdId'] = cbsd_ids[1]
+    grant_e = json.load(
+        open(os.path.join('testcases', 'testdata', 'grant_0.json')))
+    grant_e['cbsdId'] = cbsd_ids[2]
+    grant_e['operationParam']['operationFrequencyRange']\
+                             ['lowFrequency'] = 3650000000
+    grant_e['operationParam']['operationFrequencyRange']\
+                             ['highFrequency'] = 3660000000
+    request = {'grantRequest': [grant_a, grant_c, grant_e]}
+    # send grant request
+    response = self._sas.Grant(request)['grantResponse']
+    grant_ids = []
+    grant_expire_times = []
+    for response_num, resp in enumerate(response):
+      self.assertEqual(resp['response']['responseCode'], 0)
+      grant_ids.append(resp['grantId'])
+      grant_expire_times.append(
+          datetime.strptime(resp['grantExpireTime'], '%Y-%m-%dT%H:%M:%SZ'))
+    del request, response
+    # First Heartbeat
+    request = []
+    for cbsd_id, grant_id in zip(cbsd_ids, grant_ids):
+        request.append({
+            'cbsdId': cbsd_id,
+            'grantId': grant_id,
+            'operationState': 'GRANTED'
+        })
+    
+    response = self._sas.Heartbeat({'heartbeatRequest': request})['heartbeatResponse']
+    
+    del request, response
+    # STEP 2
+    # load and inject FSS data with Overlapping Frequency of CBSD
+    fss_e = json.load(
+        open(os.path.join('testcases', 'testdata', 'fss_record_0.json')))
+    self._sas_admin.InjectFss({'record': fss_e})   
+    # load and inject GWPZ data with Overlapping Frequency of CBSD 
```

Comment:
Technically this should be GWBL data. I think you can just change "gwpz" to "gwbl" here and in the name of the file to avoid confusion.